.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,76 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+# Run on:
+# - pushes to master, or
+# - PRs with a base of `master`
+# - which do not **only** consist of changes to .md or .less files
+on:
+  push:
+    branches: [ master ]
+    paths-ignore:
+      - '**/*.md'
+      - '**/*.less'
+  pull_request:
+    branches: [ master ]
+    paths-ignore:
+      - '**/*.md'
+      - '**/*.less'
+  schedule:
+    - cron: '0 0 * * 1,3,5'
+
+jobs:
+  analyze:
+    name: Analyze / ${{ matrix.language }}
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'javascript' ]
```

Comment:
Should typescript be listed too? I'm assuming its included in `javascript`,

Reply:
It is :)